csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -738,7 +752,7 @@ private void ResolveConflicts(IEnumerable<string> frameworkPaths)
 
         private void AnalyseSolutions(IEnumerable<string> solutions)
         {
-            Parallel.ForEach(solutions, new ParallelOptions { MaxDegreeOfParallelism = options.Threads }, solutionFile =>
+            Parallel.ForEach(solutions, new ParallelOptions { MaxDegreeOfParallelism = EnvironmentVariables.GetDefaultNumberOfThreads() }, solutionFile =>
```

Comment:
nit: Maybe store the number of threads in a readonly field instead of accessing the environment variable multiple times throughout the code.

Reply:
Done.